netket/operator/_local_operator.py

```diff
@@ -722,11 +757,13 @@ def _multiply_operator(self, op, act):
 
             if act.size == act_i.size and np.array_equal(act, act_i):
                 # non-interesecting with same support
-                operators.append(np.copy(np.matmul(self._operators[i], op)))
+                # TODO is it necessary to create a copy here?
+                operators.append(self._operators[i].dot(op))
```

Comment:
i dont understand the todo. ( if you are referring to the old code, no, i'm pretty sure the copy was not needed).

As for this...
Are you sure that `matmul` is equivalent to `dot` in this case?

Reply:
It is equivalent. Though `@` works too. I'll change it to that notation.